app/controllers/invitations_controller.rb

```diff
@@ -0,0 +1,28 @@
+class InvitationsController < ApplicationController
+  before_action :require_user
+
+  def new
+    @invitation = Invitation.new
+  end
+
+  def create
+    @invitation = Invitation.new(invitation_params)
+    @invitation.author = current_user
```

Comment:
I disagree on 'added complexity'.  We are adding the author to the hash of params and it will build an invitation in 1 go. its still within 1 line.

On the other side, we are setting author in a separate line. Less lines generally mean less bugs.